models/user.js

```diff
@@ -8,10 +8,26 @@ module.exports = function(sequelize, DataTypes) {
     password: DataTypes.STRING,
     enable_sso: DataTypes.BOOLEAN,
     display_name: DataTypes.STRING,
-    photo_url: DataTypes.STRING
+    photo_url: DataTypes.STRING,
+    admin: DataTypes.BOOLEAN   // maybe replace that by an array of roles
   }, {
     underscored: true,
-
+    instanceMethods: {
+      setPassword: function(password, done) {
+        return bcrypt.genSalt(10, function(err, salt) {
+          return bcrypt.hash(password, salt, function(error, encrypted) {
+            this.password = encrypted;
+            this.salt = salt;
+            return done();
+          });
+        });
+      },
+      verifyPassword: function(password, done) {
+        return bcrypt.compare(password, this.password, function(err, res) {
```

Comment:
This can be simplified to `return bcrypt.compare(password, this.password, done);`